study1/week1/SetMismatch.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Hashtable;
+class Solution {
+    Hashtable<Integer, Boolean> checklist = new Hashtable<Integer, Boolean>();
+    int[] res = new int[2]; // Space Complexity O(1)
+
+    public int[] findErrorNums(int[] nums) {
+        int length = nums.length; // Space Complexity O(length)
```

Comment:
I don't know if you meant the `length` variable, but if so, it is `O(1)`
The hash map will contain `n-1` entries so it'll potentially consume `O(n)` space.